src/styles/Base/typography.scss

```diff
@@ -1,11 +1,43 @@
 /* Styles for fonts, line-height, paragraphs, h1, h2, h3 ... */
+html {
+  font-size: 1.6rem;
+  line-height: 1.5;
+}
+
 body {
   font-family: $font-text;
 }
-h1, h2, h3, h4, h5, h6 {
+
+h1, h2, h3 {
   font-family: $font-heading;
 }
 
+/* Vertical Rhythm TODO: Mixin it */
+h1 {
+  font-size: 2rem;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 1.8rem;
+  font-weight: 500;
+  line-height: 1.33;
+}
+
+h3 {
+  font-size: 1.6rem;
+  font-weight: 500;
+  line-height: 1.5;
+}
+
+h1, h2, h3, h4, h5, h6, hgroup,
+ul, ol, dd,
+p, figure,
+pre, table, fieldset, hr {
+  margin: 0.75rem 0;
+}
+
 p {
   font-family: $font-text;
```

Comment:
Good spot. Paragraphs are going to inherit the base size from the `<html>` element. On line 2 of the beginning of this file I set the size of the `html` element to be `1.6rem`. So when an element in the page doesn't have a set font-size its going to inherit the one from the `html` element.